### ISSUE TITLE
Regarding CPU utilisation on Arm compute library 

### ISSUE THREAD

**Issue by abhajaswal** [ARM-software/ComputeLibrary#966]
Hi , Already had opened this issues in https://github.com/ARM-software/ComputeLibrary/issues/950
But the tool mentioned here to profile cpu utiilisation  is using ARMNN which uses tflite as the input .

My issue is i am using a tensorflow model using graph interface for Arm compute library.

The Configurations one time for ACL is as below :

do_setup(void)
{

	common_params.target = arm_compute::graph::Target::NEON;
	common_params.data_path = "/usr/share/cl-res/";
	common_params.labels = "/usr/share/cl-res/labels.txt";

	// Get trainable parameters data path
	std::string data_path = common_params.data_path;
	ConvolutionMethod convolution_hint = ConvolutionMethod::GEMM;

	// Create input descriptor
	const TensorShape tensor_shape = permute_shape(TensorShape(64U, 96U, 1U, 1U), DataLayout::NCHW, common_params.data_layout);
	TensorDescriptor  input_descriptor = TensorDescriptor(tensor_shape, common_params.data_type).set_layout(common_params.data_layout);

	arm_compute::FullyConnectedLayerInfo fc_info; fc_info.weights_trained_layout = common_params.data_layout;
	fc_info.transpose_weights = false;

	// Set weights trained layout
	const DataLayout weights_layout = common_params.data_layout;
        const auto        operation_layout = common_params.data_layout;

	graph << common_params.target
		<< common_params.fast_math_hint
		<< convolution_hint
		<< InputLayer(input_descriptor,
		std::make_unique<AI_InputAccessor>(input_to_acl)).set_name("input0")
		<< ConvolutionLayer(
		5U, 5U, 32U,
		get_weights_accessor(data_path, "classifier_filter_conv1-acl.npy", weights_layout),
		get_weights_accessor(data_path, "classifier_bias_conv1.npy"),
		PadStrideInfo(1, 1, 0, 0))
		.set_name("conv1")
		<< ActivationLayer(ActivationLayerInfo(ActivationLayerInfo::ActivationFunction::RELU)).set_name("relu1")
                << PoolingLayer(PoolingLayerInfo(PoolingType::MAX, Size2D(2,2), operation_layout, PadStrideInfo(2, 2, 0, 0))).set_name("pool1")
		<< ConvolutionLayer(
		3U, 3U, 16U,
		get_weights_accessor(data_path, "classifier_filter_conv2-acl.npy", weights_layout),
		get_weights_accessor(data_path, "classifier_bias_conv2.npy"),
		PadStrideInfo(1, 1, 0, 0))
		.set_name("conv2")
		<< ActivationLayer(ActivationLayerInfo(ActivationLayerInfo::ActivationFunction::RELU)).set_name("relu2")
                << PoolingLayer(PoolingLayerInfo(PoolingType::MAX, Size2D(2,2), operation_layout, PadStrideInfo(2, 2, 0, 0))).set_name("pool2")
		<< ConvolutionLayer(
		3U, 3U, 16U,
		get_weights_accessor(data_path, "classifier_filter_conv3-acl.npy", weights_layout),
		get_weights_accessor(data_path, "classifier_bias_conv3.npy"),
		PadStrideInfo(1, 1, 0, 0))
		.set_name("conv3")
                << PoolingLayer(PoolingLayerInfo(PoolingType::AVG, Size2D(12,20), operation_layout, PadStrideInfo(1, 1, 0, 0))).set_name("pool3")
		<< ReshapeLayer(TensorShape(16U, 1U)).set_name("Reshape")
		<< BatchNormalizationLayer(
		get_weights_accessor(data_path, "classifier_sub_mean_BN.npy"),
		get_weights_accessor(data_path, "classifier_sub_variance_BN.npy"),
		get_weights_accessor(data_path, "classifier_sub_gamma_BN.npy"),
		get_weights_accessor(data_path, "classifier_sub_beta.npy"),
		0.000f).set_name("subBN")
		<< L2NormalizeLayer(0, 1e-12f).set_name("l2norm")
		<< FullyConnectedLayer(
		4U,
		get_weights_accessor(data_path, "classifier_linear_weight.npy", weights_layout),
		std::unique_ptr<arm_compute::graph::ITensorAccessor>(nullptr),
		fc_info).set_name("dense0")
		<< SoftmaxLayer().set_name("prob")
		<< OutputLayer(std::make_unique<ProbAccessor>(prob_out));
	// Finalize graph
	GraphConfig config;
	config.num_threads = common_params.threads;
	config.use_tuner = common_params.enable_tuner;
	config.tuner_mode = common_params.tuner_mode;
	config.tuner_file = common_params.tuner_file;
	config.use_transition_memory_manager = false;
	LOG_DEBUG("SETUP of classifier is OVER HERE \n");

	graph.finalize(common_params.target, config);
	LOG_DEBUG("graph finalize is OVER HERE \n");

	graph.run(); /// dummy run without actual input as the run time buffer allocation take lot of time

}

In case i convert tensorflow model to tflite it optimises this model and introduces depthwise convolution layer as 1st layer and other  operation changes which does not allow me one to one mapping of this ACL layer usage and tflite model profile using ARMNN 

I face a tricky issue of jump in Current for hardware board to 7 Amp when using NEON based ACL inference in my application scenario , so i was wondering if its possible for me to know the Exact CPU usage, processing /memory utilisation  using only ACL  neon kernels.
